panda/src/speedtree/speedTreeNode.cxx

```diff
@@ -145,9 +145,6 @@ count_total_instances() const {
  */
 SpeedTreeNode::InstanceList &SpeedTreeNode::
 add_tree(const STTree *tree) {
-  nassertr(is_valid(), *(InstanceList *)NULL);
-  nassertr(tree->is_valid(), *(InstanceList *)NULL);
```

Comment:
This is bad (but I'm afraid of touching the SpeedTree code because I don't have a licence to compile it with).  Some code in Panda allocates a static const version of the class to return a reference to in case of assertions.

Reply:
Resolution: `assert` (and add TODO comment saying this needs to be nassertr but there's no good, safe, return type so we have to use assert for now)